brickflow/codegen/databricks_bundle.py

```diff
@@ -1050,6 +1051,11 @@ def proj_to_bundle(self) -> DatabricksAssetBundles:
                 parameters=workflow.parameters,
                 environments=workflow.environments,
                 git_source=git_conf,
+                queue=(
+                    JobsQueue(enabled=workflow.queue)
+                    if workflow.queue is not None
```

Comment:
The way this is set up, it will evaluate to True even if this is set to False (False is not None) - is that intended? I think a Boolean value would be better here. Default False (backwards compatible + non breaking)

Reply:
this should work as intended. There are three states, a user does not define anything (server default which is true). user decides true, user decides false.

```
if __name__ == "__main__":

    from dataclasses import dataclass

    @dataclass
    class JobQueue:
        enabled: bool

    values = [None, True, False]
    for value in values:
        print(f"Value is {value}")
        this_val = JobQueue(enabled=value) if value is not None else None
        print(f"This val is {this_val}")
```

```
Value is None
This val is None
Value is True
This val is JobQueue(enabled=True)
Value is False
This val is JobQueue(enabled=False)
```